docs/guides/template-locations.md

```diff
@@ -23,6 +23,31 @@ Timber::$locations = array(
 );
 ```
 
+You can also use namespaces in your locations too, just define it as the value next to a path, for example:
+
+```php
+<?php
+Timber::$locations = array(
+    '/Users/jared/Sandbox/templates',
+    '~/Sites/timber-templates/',
+    ABSPATH.'/wp-content/templates',
+    ABSPATH.'/wp-content/styleguide' => 'styleguide'
+);
+```
+
```

Comment:
Here I would add an example for how this can be used in Twig with the `@` character, pretty much in the same way that the [Symfony Docs](https://symfony.com/doc/current/templating/namespaced_paths.html#registering-your-own-namespaces) do it.

Reply:
Addressed in https://github.com/timber/timber/pull/1791/commits/9abb875f985827b741fc50006408848253c02c06

Reply:
Cool! 👍